examples/solid mechanics/viscoplastic-damage/beam/beam.json

```diff
@@ -37,7 +37,7 @@
     "Mesh" : [{
             "Name" : "beam",
             "ConstitutiveModel" : {
-                "Name" : "ChabocheDamage", //"ChabocheDamage", J2Plasticity
+                "Name" : "ChabocheDamage",
```

Comment:
This doesn't really give any details that J2 plasticity is being used as the constitutive model.  We could change this to an extra parameter
```json
"Name" : "J2Plasticity",
"Damage" : "Chaboche"
```
or something similar.

Reply:
"J2PlasticityDamage" should be fine so we don't have an extra parameter. I'll include it in the next commit.